org.musescore.MuseScore.appdata.xml

```diff
@@ -92,7 +92,7 @@
   </screenshots>
   <content_rating type="oars-1.1"/>
   <releases>
-    <release date="2018-07-31" version="2.3.2">
+    <release date="2018-12-07" version="3.0beta2">
```

Comment:
Don’t rewrite the old release notes tag with the new one… not only we’ll lose the old release notes, the new ones will be wrong.